### PR TITLE
Update angular-js.md

### PR DESCRIPTION
**Pull request by coderNadir**
Added the tutorial of: how to integrate Web3Forms API in Angular step by step.

### CODE REVIEW

File: how-to-guides/js-frameworks/angular-js.md [hunk 1]
```diff
@@ -1,6 +1,126 @@
...
+- that's it for the _**<u>mail service</u>**_.
+
+---
+
+- Now let's take a look at the HTML **_<u>form</u>_**:
+_We splitted the Form HTML code into 2 images for a better view as well as we are you using TailwindCSS for styling_
+
+<img width="668" alt="angular HTML form part 1" src="https://github.com/surjithctly/web3forms-docs/assets/78637183/9bf71113-daff-4b33-8a0f-4328f4b99a17"> 
+
+<img width="668" alt="angular HTML form part 2" src="https://github.com/surjithctly/web3forms-docs/assets/78637183/1245cccc-3fb9-4eed-b9b7-134f0ae36515">
```
Comment (surjithctly) [inline how-to-guides/js-frameworks/angular-js.md]: Can you use code ``` blocks instead of images? so people can copy paste easily.

**Comment by surjithctly** [comment]
Also, remove the create access key from web3forms & web3forms as pre-requesties etc. Since its a docs, we already expect people to be familiar with web3forms.

**Comment by coderNadir** [comment]
I've modified what you asked me to modify and made a bit changes too.

**Review by surjithctly — CHANGES_REQUESTED** [review]
Can you remove image screenshots of code and add it as a copy pastable code blocks? 

```
like this
```

File: how-to-guides/js-frameworks/angular-js.md (hunk 1 above)
Comment (coderNadir) [reply how-to-guides/js-frameworks/angular-js.md]: done replacing the screenshots with actual code to enable copying code

**Pull request merged**